url.json

```diff
@@ -1,7 +0,0 @@
-[
-    {
-      "name": "lambdatest-home-page",
-      "url": "https://www.lambdatest.com"
-    }
-  ]
-  
```

Comment:
@pinanks `smartui-web.json` and `url.json` file was added purposefully, please add that